command/meta.go

```diff
@@ -50,10 +50,9 @@ type Meta struct {
 	// for some reason.
 	OriginalWorkingDir string
 
-	Color            bool             // True if output should be colored
-	GlobalPluginDirs []string         // Additional paths to search for plugins
-	PluginOverrides  *PluginOverrides // legacy overrides from .terraformrc file
```

Comment:
Was this just dead code?

Reply:
Yeah... we were still populating it in `package main` before, but nothing was actually reading from it. I deleted it here because it seemed confusing with the new thing I was adding which also included the word "overrides". :smile: